src/scss/components/_scenarios.scss

```diff
@@ -1,7 +1,15 @@
+* {
+  margin: 0 auto;
+  max-width: 450px;
+}
 .scenarios {
   padding-bottom: 230px;
+  padding-left: 30px;
+  padding-right: 30px;
   padding-top: 30px;
 
+  @extend .container;
```

Comment:
You don't need this because you are already adding the class `container` in the JS

Reply:
Removed